plugins/workspace-minimap/src/positioned_minimap.ts

```diff
@@ -127,6 +135,17 @@ export class PositionedMinimap extends Minimap implements Blockly.IPositionable
       this.setMinimapCss();
     }
 
+
+    /**
+     * Sizes the minimap.
+     */
+    size(): void {
+      const viewWidth = this.primaryWorkspace.getMetrics().viewWidth;
+      this.width = Math.max(minWidth, viewWidth / 5);
+      this.height = this.width / 1.5;
```

Comment:
Where are these values getting applied?

Reply:
In the `setMinimapCss` method (which is called in the position method and when the window resizes)